tests/test_utils_python.py

```diff
@@ -239,8 +239,11 @@ def __call__(self, a, b, c):
         self.assertEqual(get_func_args(" ".join, stripself=True), ["iterable"])
 
         if platform.python_implementation() == "CPython":
-            # doesn't work on CPython: https://bugs.python.org/issue42785
-            self.assertEqual(get_func_args(operator.itemgetter(2)), [])
+            # Didn't used to work on CPython: https://bugs.python.org/issue42785
```

Comment:
I'd rephrase it: https://english.stackexchange.com/questions/8816/whats-the-negation-of-i-used-to-be-surely-not-i-didnt-used-to-be

Reply:
Updated